tests/foreman/ui/test_contenthost.py

```diff
@@ -135,11 +137,30 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    rhel_lifecycle_status = 'Unknown'
+    _distro = module_repos_collection_with_manifest.distro
+    if _distro == 'rhel7':
+        rhel_lifecycle_status = 'Approaching end of maintenance support'
+    elif _distro == 'rhel8':
+        rhel_lifecycle_status = 'Full support'
```

Comment:
Do we have another way to check the current support status so that we don't need to hardcode it?

Reply:
will look into this soon once I can get passing

Reply:
Would you say the two new helper function address this @vsedmik ?

Reply:
Not sure if I read it correctly, this way we grant last two (numerical) entries in the yaml as `Full support`, and entries left lower and lower support level?
Proposal: Let's not depend on the position (though I don't expect it would change to anything like `[8,9,7,6]`, rendering 6 and 7 as `Full support`) but rather it's value like this.
```
        rhels = get_supported_rhel_versions().sort(reverse=True)
        rhel_lifecycle_status = 'Unknown'
        if rhel_version not in rhels:
            return rhel_lifecycle_status
        elif rhels.index(rhel_version) <= 1:
            rhel_lifecycle_status = 'Full support'
        elif rhels.index(rhel_version) == 2:
            rhel_lifecycle_status = 'Approaching end of maintenance support'
        elif rhels.index(rhel_version) >= 3:
            rhel_lifecycle_status = 'End of maintenance support'
        return rhel_lifecycle_status
```